dom.bs

```diff
@@ -939,6 +946,50 @@ when something has occurred.
 <a for=/>list</a> of zero or more <a>event listeners</a>). It is initially the empty list.
 <!-- Intentionally not exported. -->
 
+<p>Each {{EventTarget}} object has an associated <dfn for=EventTarget>attached internals</dfn> (null
+or an {{EventTargetInternals}} object), initially null.
+
+<p>Each {{EventTargetInternals}} object has an associated <dfn
+for=EventTargetInternals>eventTarget</dfn> (an {{EventTarget}} object).
+
+<p>The <dfn export for=Event id=concept-eventtarget-constructor>new EventTarget(cb)</dfn> constructor steps are:
+
+<ol>
+ <li><p>If <var>cb</var> is not null then:
+
+  <ol>
+   <li><p>Let <var>eventTargetInternals</var> a new {{EventTargetInternals}} instance.
+
+   <li><p>Set <var>eventTargetInternals</var>'s <a for=EventTargetInternals>eventTarget</a> to
+   <var>this</var>.
+
+   <li><a>Invoke</a> <var>cb</var> with « <var>eventTargetInternals</var> » and with <var>this</var>
+   as the <a>callback this value</a>.
+
+   <li><p>Set <var>this</var>'s <a for="EventTarget">attached internals</a> to
+   <var>eventTargetInternals</var>.
+  </ol>
+</ol>
+
+<p>To <dfn export for=Event id=concept-eventtargetinternals-set-parent>set the parent</dfn> given
+an {{EventTargetInternals}} <var>internals</var> and {{EventTarget}} <var>theParent</var>:
+
+<ol>
+  <li>If <var>theParent</var> is equal to <var>internals</var>' <a
+  for=EventTargetInternals>eventTarget</a> then <a>throw</a> a
+   "{{HierarchyRequestError!!exception}}" {{DOMException}}.
+
+  <li>If <var>theParent</var>'s <a for=EventTarget>attached internals</a>
+  {{EventTargetInternals/parent}} is equal to <var>internals</var>' <a
+  for=EventTargetInternals>eventTarget</a> then<a>throw</a> a "{{HierarchyRequestError!!exception}}"
+  {{DOMException}}.
+
+  <li>Set <var>internal</var>'s <a for=EventTargetInternals>eventTarget</a> <a>get the parent</a>
```

Comment:
I couldn't find precedent for setting an algorithm so this is just made up, hopefully it suffices?